webapp/components/test-search.html

```diff
@@ -38,7 +38,128 @@
     </div>
   </template>
 
+  <script type="text/javascript" src="https://unpkg.com/ohm-js@latest/dist/ohm.min.js"></script>
+  <script type="text/ohm-js">
```

Comment:
What's with the extra `script` tags?

Reply:
OHM has a mechanism for scraping the text of a script tag. I originally was using it, but it doesn't play well with web components, so I switched to a multiline string. This is just leftover; I'll delete it.